spec/plugin_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'plugin' do
+  before { SiteSetting.url_filters_enabled = true }
+
+  describe 'Topic Queries' do
+    describe 'after_or_before_date' do
+      before do
+        @topic = Fabricate(:topic, created_at: 10.days.ago)
+        @topic2 = Fabricate(:topic, created_at: 8.days.ago)
+        @topic3 = Fabricate(:topic, created_at: 6.days.ago)
+        @user = Fabricate(:user)
+      end
```

Comment:
```suggestion
      fab!(:topic) { Fabricate(:topic, created_at: 10.days.ago) }
      fab!(:topic2) { Fabricate(:topic, created_at: 8.days.ago) }
      fab!(:topic3) { Fabricate(:topic, created_at: 6.days.ago) }
      fab!(:user) { Fabricate(:user) }
```

(references to those objects will have to be be changed, e.g. `@user` -> `user`)